### DISCUSSION TITLE
戰爭中的 AI：烏克蘭用臉部辨識俄羅斯士兵，但這只是小兒科

### DISCUSSION THREAD

**Discussion by Jimmy911218** [1101-GS4719-C/Python-Community-GS4719-C#506] [General]
現在烏、俄戰爭越演越烈，竟然也把ai用在了戰爭上。
讓AI成了另一種殺人工具，令人反思啊!
https://technews.tw/2022/03/21/ai-in-war/

**Comment by chupochun** [comment]
AI是很方便的工具，但是用在戰爭上面實在是令人不齒

**Comment by yucheng1028** [comment]
如果有一天AI有自己的意識，按下核彈發射鈕，抖抖抖抖

**Comment by ghost** [comment]
用在戰爭也沒什麼不齒吧，畢竟這也是一種技術應用，不過，確實我覺得最後倫理道德的問題會收束在人格權之類的議題上

**Comment by sean1325418** [comment]
科技這樣發展也只能怪人類的天性，總是有人想挑起戰爭

**Comment by ThomasMeng0515** [comment]
AI真的已經無所不在，相關的道德議題也要妥善規定才是

**Comment by HuanYuuuuuu** [comment]
原來人臉辨識也可以用在這種地方.....

**Comment by hapiwangy** [comment]
我之前看過相關報導，內容是說希望可以用AI的技術來戳破俄羅斯人大外宣的目的，並給俄羅斯軍人的家屬一個交代。我想這應該是可以接受的應用吧

**Comment by cordydeng** [comment]
人臉辨識如果被有心人士利用的話可能會造成隱私權相關的問題

**Comment by Sazmafuji** [comment]
AI科技用在戰爭有些傷心，希望未來可以用AI去解決戰爭問題

**Comment by HaoFish123** [comment]
感謝提供AI相關的資訊，希望戰爭快點結束。

**Comment by xkllkx** [comment]
謝謝分享，希望戰爭趕快平息，讓民眾早日脫離苦海。

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，AI很方便，但也希望戰爭能盡快結束。

**Comment by bloodtrail0511** [comment]
這也算是AI其中一種應用，但這種應用是絕大多數人都不願看見的，希望戰爭早日結束

**Comment by HuanYuuuuuu** [comment]
深偽技術用在資訊情報戰上感覺也會相當的可怕
